inference-engine/tests/functional/plugin/cpu/subgraph_tests/src/conv_dw_conv_fusing.cpp

```diff
@@ -0,0 +1,121 @@
+
+    size_t numOutChannels, dwStride;
+    bool withBias, withDWBias;
+    std::tie(numOutChannels, dwStride, withBias, withDWBias) = convDWConvPatternParams;
+
+    result << "O=" << numOutChannels << "_";
+    result << "dwStride=" << dwStride << "_";
+    result << "withBias=" << withBias << "_";
+    result << "withDWBias=" << withDWBias;
+
+    return result.str();
+}
+
+void ConvDWConvFusingSubgraphTest::CheckConvCount() {
+    InferenceEngine::CNNNetwork execGraphInfo = executableNetwork.GetExecGraphInfo();
+    auto function = execGraphInfo.getFunction();
+    ASSERT_NE(nullptr, function);
+    size_t actualConvCount = 0;
+    for (const auto &node : function->get_ops()) {
+        const auto & rtInfo = node->get_rt_info();
+        auto getExecValue = [&rtInfo](const std::string & paramName) -> std::string {
+            auto it = rtInfo.find(paramName);
+            IE_ASSERT(rtInfo.end() != it);
+            auto value = std::dynamic_pointer_cast<ngraph::VariantImpl<std::string>>(it->second);
+            IE_ASSERT(nullptr != value);
+            return value->get();
+        };
+        if (getExecValue(ExecGraphInfoSerialization::LAYER_TYPE) == "Convolution") {
+            actualConvCount++;
+        }
+    }
+
+    ASSERT_EQ(expectedConvCount, actualConvCount);
+}
+
+bool ConvDWConvFusingSubgraphTest::isFusedConvExpected(const ngraph::Shape& inShape, const ngraph::Shape& outShape, const size_t elemSize) {
+    if (inShape.size() != 4 || outShape.size() != 4)
+        return false;
+
+    if (!InferenceEngine::with_cpu_x86_avx2() || InferenceEngine::with_cpu_x86_avx512f())
+        return false;
+
+    int L3_cache_size = getCacheSize(3, false);
+    int dw_conv_input_size = inShape[0] * inShape[1] * inShape[2] * inShape[3] * elemSize;
+    int dw_conv_output_size = outShape[0] * outShape[1]* outShape[2] * outShape[3] * elemSize;
+
+    return (dw_conv_input_size + dw_conv_output_size > L3_cache_size / 2);
+}
+
+void ConvDWConvFusingSubgraphTest::SetUp() {
+    targetDevice = CommonTestUtils::DEVICE_CPU;
+    ConvDWConvPatternParams convDWConvPatternParams;
+    SizeVector inputShape;
+
+    std::tie(convDWConvPatternParams, inputShape) = this->GetParam();
+    size_t numOutChannels, dwStride;
+    bool withBias, withDWBias;
+
+    std::tie(numOutChannels, dwStride, withBias, withDWBias) = convDWConvPatternParams;
+
+    auto inputParams = ngraph::builder::makeParams(ngraph::element::f32, {inputShape});
+    auto paramOuts = ngraph::helpers::convert2OutputVector(ngraph::helpers::castOps2Nodes<ngraph::op::Parameter>(inputParams));
+
+    auto conv1 = ngraph::builder::makeConvolution(paramOuts[0], ngraph::element::f32, {1, 1}, {1, 1}, {0, 0},
+                                                  {0, 0}, {1, 1}, ngraph::op::PadType::EXPLICIT, numOutChannels, withBias);
+
+    auto conv2 = ngraph::builder::makeGroupConvolution(conv1, ngraph::element::f32, {3, 3}, {dwStride, dwStride}, {1, 1}, {1, 1}, {1, 1},
```

Comment:
We also need to cover:
1) Cases with non fp32 prevision.
2) Cases when Conv and DW conv have other fusings: Eltwise/Depthwise/Sum.